magda-web-client/src/Components/Common/TooltipWrapper.tsx

```diff
@@ -98,10 +90,16 @@ class TooltipWrapper extends React.Component<Props, State> {
         }
     }
 
+    forceOpen = () => {
+        this.setState({
+            open: true
+        });
+    };
+
     render() {
         const className = this.props.className ? this.props.className : "";
-        const openClass =
-            this.props.startOpen && !this.state.dismissed ? "tooltip-open" : "";
+        console.log(this.state.open);
```

Comment:
Good job to make it work on mobile 👍 
Tested on my iPhone and works well 💯 
only one tiny thing is this `console.log` line, if we don't need it, we probably want it removed?

Reply:
oops :|